enhance-windows-ci-builds.feature

```diff
@@ -0,0 +1,3 @@
+Add new Windows build agent (disabled for now), and disable a different broken agent.
```

Comment:
Release notes are needed for user-visible features. In this case the user doesn't see anything yet, is that right?

Reply:
Oh. I just followed the PR template. I can remove this if you'd like me to.

Reply:
In this case I will need to remove this release note when doing the release anyway, so better remove.

I should have said something about PRs with non user visible features in the PR template

Reply:
OK, removed in 21eae23e.